pages/resultset.py

```diff
@@ -30,9 +32,12 @@ def open_next_unclassified_failure(self):
         self.selenium.find_element(*self._resultset_locator).send_keys("n")
 
     def open_logviewer(self):
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._result_status_locator).is_displayed())
+        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.selenium.find_element(*self._job_details_actionbar_locator).is_displayed())
         self.selenium.find_element(*self._resultset_locator).send_keys("l")
 
+    def return_unclassified_failure_count(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).int(element.text)
```

Comment:
Also, this is trying to call an `int` method on the `WebElement` that doesn't exist. Also, you reference an `element` variable that doesn't exist. To cast a string to an integer you would need to wrap the entire call in `int()`. For example:

```python
return int(self.selenium.find_element(*self._unclassified_failure_count_locator).text)
```